### ISSUE TITLE
Numberpad keys don't jump to the selected screen

### ISSUE THREAD

**Issue by mikeclayton** [mikeclayton/FancyMouse#39]
This code doesn't work properly when using numberpad keys to jump to a screen from the preview image:

https://github.com/mikeclayton/FancyMouse/blob/0cf930d7d044df52ac74393827016c84755a901e/src/FancyMouse/UI/FancyMouseForm.cs

```
 case (>= Keys.D1 and <= Keys.D9) or (>= Keys.NumPad1 and <= Keys.NumPad9): 
     // number keys 1-9 or numpad keys 1-9 - move to the numbered screen 
     var screenNumber = e.KeyCode - Keys.D0; 
     if (screenNumber <= screens.Count) 
     { 
         targetScreenNumber = screenNumber; 
     } 
```

We should handle number keys separate from numberpad keys, and subtract ```Keys.NumPad0``` for numberpad keys, not ```Keys.D0``` - numberpad keys don't actually do anything at the moment as a result...

**Comment by mikeclayton** [comment]
Fixed in #40

**Comment by mikeclayton** [comment]
Fixed in #40 / [FancyMouse v0.0.18-preview](https://github.com/mikeclayton/FancyMouse/releases/tag/v0.0.18-preview)
